src/commands/create_manifest.ts

```diff
@@ -0,0 +1,67 @@
+import { EID } from 'ardrive-core-js';
+import { cliArDriveFactory, cliWalletDao } from '..';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import { SUCCESS_EXIT_CODE } from '../CLICommand/error_codes';
+import {
+	BoostParameter,
+	FolderIdParameter,
+	DriveIdParameter,
+	DryRunParameter,
+	SeedPhraseParameter,
+	TreeDepthParams,
+	WalletFileParameter,
+	DestinationManifestNameParameter
+} from '../parameter_declarations';
+
+new CLICommand({
+	name: 'create-manifest',
+	parameters: [
+		DriveIdParameter,
+		FolderIdParameter,
+		DestinationManifestNameParameter,
+		BoostParameter,
+		DryRunParameter,
+		WalletFileParameter,
+		SeedPhraseParameter,
+		...TreeDepthParams
+	],
+	action: new CLIAction(async function action(options) {
+		const parameters = new ParametersHelper(options, cliWalletDao);
+
+		const wallet = await parameters.getRequiredWallet();
+
+		const arDrive = cliArDriveFactory({
+			wallet: wallet,
+			feeMultiple: parameters.getOptionalBoostSetting(),
+			dryRun: !!options.dryRun
+		});
+
+		// User can specify either a drive ID or a folder ID
+		const driveId = parameters.getParameterValue(DriveIdParameter);
```

Comment:
Consider updating `getParameterValue` to behave like `getRequiredParameterValue`, i.e. take a function that can map the value to the correct type. In this instance, we'd be using `EID` as the function.

Reply:
NOTE: Be careful with optionals